cmd/subcmds/output.go

```diff
@@ -84,3 +100,30 @@ func pickWriter(args *inArgs, data *bytes.Buffer) (ir.Writer, error) {
 		return nil, fmt.Errorf("bad output format: %q", args.outputFmt)
 	}
 }
+
+func writeLocals(args *inArgs, collection ir.Collection, defs *ir.ConfigDefs) error {
+	var data *bytes.Buffer
+	var err error
+
+	if !args.locals {
+		return nil
+	}
+	if args.outputFmt != tfOutputFormat {
+		return fmt.Errorf("--locals flag can be supplied only when the output format is tf")
+	}
+
+	_, isACLCollection := collection.(*ir.ACLCollection)
+	if data, err = tfio.WriteLocals(defs, isACLCollection); err != nil {
+		return err
+	}
```

Comment:
```suggestion
	if !args.locals {
		return nil
	}
	if args.outputFmt != tfOutputFormat {
		return fmt.Errorf("--locals flag can be supplied only when the output format is tf")
	}

	_, isACLCollection := collection.(*ir.ACLCollection)
	data, err := tfio.WriteLocals(defs, isACLCollection)
	if err != nil {
		return err
	}
```
Declare variables as close as possible to the place where they are being used